### ISSUE TITLE
Discrepancy in conversion of output to logits.

### ISSUE THREAD

**Issue by krantiparida** [swimmiing/ACL-SSL#3]
I noticed there is a difference in the way the outputs are converted into logits for training and evaluation. [Here](https://github.com/swimmiing/ACL-SSL/blob/bd506da50c5c3ca86820a93e69e2897439f72290/modules/mask_utils.py#L64) the logits are obtained by multiplying with `w` and adding `b` to it during training. But during [evaluation](https://github.com/swimmiing/ACL-SSL/blob/bd506da50c5c3ca86820a93e69e2897439f72290/modules/mask_utils.py#L113), the operation is different, where the output is added to `b/w`.  Can you please clarify it? Thanks in advance.

**Comment by swimmiing** [comment]
That part can be considered a method for finding the optimal threshold for heatmap binarization during testing. Typically, the slope of the sigmoid function σ(wx+b) becomes sharp during training. By smoothing the slope of the sigmoid activation, this approach makes it easier to find the optimal threshold even at a less precise resolution (We use 0.05 resolution in Eval.py).

**Comment by krantiparida** [comment]
Thanks for the clarification.
